androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/surveydetail/SurveyStartContent.kt

```diff
@@ -0,0 +1,160 @@
+package co.nimblehq.avishek.phong.kmmic.android.ui.screen.surveydetail
+
+import androidx.activity.compose.BackHandler
+import androidx.compose.animation.*
+import androidx.compose.animation.core.animateFloatAsState
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.*
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.scale
+import androidx.compose.ui.graphics.*
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import co.nimblehq.avishek.phong.kmmic.android.R
+import co.nimblehq.avishek.phong.kmmic.android.ui.common.BackButton
+import co.nimblehq.avishek.phong.kmmic.android.ui.common.PrimaryButton
+import co.nimblehq.avishek.phong.kmmic.android.ui.theme.ApplicationTheme
+import co.nimblehq.avishek.phong.kmmic.presentation.uimodel.SurveyUiModel
+import coil.compose.AsyncImage
+import kotlinx.coroutines.*
+
+private const val TopGradientAlpha: Float = 0.01f
+private const val BottomGradientAlpha: Float = 0.6f
+private const val InitialImageScale: Float = 1f
+private const val FinalImageScale: Float = 1.5f
+const val ImageScaleAnimationDurationInMillis = 700
+
+@Composable
+fun SurveyStartContent(
+    surveyUiModel: SurveyUiModel,
+    onStartClick: () -> Unit,
+    onBackClick: () -> Unit,
+) {
+    var shouldShowContent by remember { mutableStateOf(false) }
+    var imageScale by remember { mutableStateOf(InitialImageScale) }
+    val scope = rememberCoroutineScope()
+
+    val backNavigationHandler: suspend CoroutineScope.() -> Unit = {
+        imageScale = InitialImageScale
+        shouldShowContent = false
+        delay(ImageScaleAnimationDurationInMillis.toLong())
+        onBackClick()
+    }
+
+    Box(
+        modifier = Modifier
+            .fillMaxSize()
+    ) {
```

Comment:
I don't think we need to go to the new line for this.
```suggestion
       Box(modifier = Modifier.fillMaxSize()) {
```

Reply:
Updated in https://github.com/nimblehq/avishek-phong-kmm-ic/pull/118/commits/1a902df4fd37f70ff2d375c7321a16fc067846ee